Add try catch for SocketException on SendTo socket

Farrser opened the pull request:
Add try catch for SocketException on SendTo socket that passes the event to the OnUnhandledException method so that it can be handled by the client appropriately

Commit:
Add try catch for SocketException on SendTo socket that passes the event to the OnUnhandledException method so that it can be handled by the client appropriately

Commit:
Merge branch 'develop' of https://github.com/owaits/ACN into develop

owaits commented on LXProtocols.TCNet/Sockets/TCNetSocket.cs:
```diff
@@ -467,7 +467,14 @@ protected void Broadcast(Socket socket, TCNetPacket packet)
+            }
+            catch (SocketException ex)
+            {
+                OnUnhandledException(new ApplicationException("Socket error broadcasting PioneerDJ integration.", ex));
```
This should say TCNet as in theory the protocol allows for implementation with other manufacturers.

owaits commented on LXProtocols.TCNet/Sockets/TCNetSocket.cs:
```diff
@@ -501,7 +508,15 @@ protected void Unicast(TCNetPacket packet, TCNetEndPoint address)
+            }
+            catch (SocketException ex)
+            {
+                OnUnhandledException(new ApplicationException("Socket error unicast PioneerDJ integration.", ex));
```
Same here, this should say TCNet.

Commit:
Change Pioneer speicific wording to TCNet

Technically the implentation of TCNet is not speicific to pioneer and so we shouldn't be giving error messages that mention pioneer. Instead we refer to the generic TCNet implentation
